Neuer Artikel zu Ausweispflichten bei Auskunftsanfragen (#282)
datenanfragen/website: content/blog/ausweispflichten.de.md

{
    "title": "Wie muss ich mich bei einer Datenschutzanfrage ausweisen?",
    "slug": "ausweispflichten",
    "date": "2020-04-02T14:00:00+02:00",
    "type": "blog",

>     "description": "Wenn Nutzer_innen eine Datenschutzanfrage stellen, werden sie häufig vom Diensteanbieter häufig aufgefordert sich auszuweisen. Grundsätzlich müssen sich Betroffene nur mit solchen Informationen ausweisen, die das Unternehmen für die eindeutige Identifizierung benögtigt.",

Reviewer: ```suggestion
    "description": "Wenn Nutzer_innen eine Datenschutzanfrage stellen, werden sie häufig vom Diensteanbieter aufgefordert sich auszuweisen. Grundsätzlich müssen sich Betroffene nur mit solchen Informationen ausweisen, die das Unternehmen für die eindeutige Identifizierung benögtigt.",
```

## Dienste bei denen sich die Nutzer_in angemeldet hat

Grundsätzlich gilt immer, dass die Anbieter_in der Nutzer_in die Ausübung seiner_ihrer Rechte nicht unmöglich gemacht werden darf. Die Anbieter_in kann sich nur dann einer Auskunft verweigern, wenn er_sie glaubhaft macht, dass sie nicht in der Lage ist, den Betroffene_n zu identifizieren, Art. 12 II 2 DSGVO. Identifizierung meint in diesem Kontext, dass die Anbieter_in sicherstellt, dass es wirklich die Nutzer_in ist, die für den entsprechenden Account Datenschutzrechte geltend macht. Es ist nicht nötig, dass die Anbieter_in den Account einer natürlichen Person zuordnen kann. Konkret reicht es, wenn die Nutzer_in nachweist, dass ihm_ihr der Account gehört,[^1] beispielsweise durch die hinterlegte Mailadresse, Benutzername und Kundennummer. In keinem Fall sollten Passwörter weitergegeben werden.

## Ausweiskopien nur in Ausnahmefällen notwendig

> Ausweiskopien sind hauptsächlich in den Fällen erforderlich, in denen die Nutzer_in Rechte gegen eine_n Verantwortliche_n geltend machen möchte, mit dem er_sie zuvor noch nicht selbst in Kontakt stand. In diesen Fällen hat die Verantwortliche meist keine andere Möglichkeit die Nutzer_in zu identifizieren. Klassischerweise sind das Situationen in denen die Rechte gegen eine Auskunftei gerichtet sind. Entgegen häufig anderslautender Gerüchte, ist die Ausweisinhaber_in berechtigt, eine Kopie seines_ihres Personalausweises zu erstellen, § 20 II PAuswG. Auch hier sind nur diejenigen Informationen erforderlich, die es der Anbieter_in erlauben, die Person, die den Anspruch geltend macht, eindeutig dem vorhandenen Datensatz zuzuordnen. Nicht erforderlich sind in aller Regel Ausweisnummer, Lichtbild, persönliche Merkmale und Staatsangehörigkeit. Die Anbieter_in darf die Ausweiskopie nur dazu nutzen, die Identität zu überprüfen. Danach muss die Kopie wieder vernichtet werden.[^2] Es empfiehlt sich, die nicht benötigten Ausschnitte des Ausweises bereits beim Scannen abzudecken. So läuft man später nicht Gefahr, dass Daten aus einem retuschierten Bild doch wieder herstellbar sind. Die verantwortliche Datenverarbeiter_in ist dazu verpflichtet, der Nutzer_in in der Wahrnehmung seiner Rechte aktiv zu unterstützen. Daraus folgt, dass die Anbieter_in den Identitätsnachweis nicht unter einem Medienbruch verlangen kann. Handelt es sich um eine internetbasierte Plattform, dürfen auch die Identitätsnachweise online erbracht werden (z.B. per Mail).

Reviewer: ```suggestion
Ausweiskopien sind hauptsächlich in den Fällen erforderlich, in denen die Nutzer_in Rechte gegen eine_n Verantwortliche_n geltend machen möchte, mit dem er_sie zuvor noch nicht selbst in Kontakt stand. Außerdem müssen "**begründete Zweifel**" [^2] an der Identität der Nutzer_in herrschen. In diesen Fällen hat die Verantwortliche meist keine andere Möglichkeit die Nutzer_in zu identifizieren. Klassischerweise sind das Situationen in denen die Rechte gegen eine Auskunftei gerichtet sind. Entgegen häufig anderslautender Gerüchte, ist die Ausweisinhaber_in berechtigt, eine Kopie seines_ihres Personalausweises zu erstellen, § 20 II PAuswG. Auch hier sind nur diejenigen Informationen erforderlich, die es der Anbieter_in erlauben, die Person, die den Anspruch geltend macht, eindeutig dem vorhandenen Datensatz zuzuordnen. Nicht erforderlich sind in aller Regel Ausweisnummer, Lichtbild, persönliche Merkmale und Staatsangehörigkeit. Die Anbieter_in darf die Ausweiskopie nur dazu nutzen, die Identität zu überprüfen. Danach muss die Kopie wieder vernichtet werden.[^3] Es empfiehlt sich, die nicht benötigten Ausschnitte des Ausweises bereits beim Scannen abzudecken. So läuft man später nicht Gefahr, dass Daten aus einem retuschierten Bild doch wieder herstellbar sind. Die verantwortliche Datenverarbeiter_in ist dazu verpflichtet, der Nutzer_in in der Wahrnehmung seiner Rechte aktiv zu unterstützen. Daraus folgt, dass die Anbieter_in den Identitätsnachweis nicht unter einem Medienbruch verlangen kann. Handelt es sich um eine internetbasierte Plattform, dürfen auch die Identitätsnachweise online erbracht werden (z.B. per Mail).
```

[^1]: Jahresbericht des Berliner Datenschutzbeauftragten, S. 132, https://www.datenschutz-berlin.de/fileadmin/user_upload/pdf/publikationen/jahresbericht/BlnBDI-Jahresbericht-2017-Web.pdf.

> [^2]: Beifügung einer Personalausweiskopie bei Auskunftsrechten, Bundesdatenschutzbeauftragter, https://www.bfdi.bund.de/SharedDocs/Publikationen/AusweiskopieAuskunftsersuchen.html.

Reviewer: ```suggestion
[^2]: Personalausweis und Datenschutz, LfDI NRW, https://www.ldi.nrw.de/mainmenu_Aktuelles/Inhalt/Personalausweis-und-Datenschutz/Datenschutz-und-Personalausweis-2019_07.pdf
[^3]: Beifügung einer Personalausweiskopie bei Auskunftsrechten, Bundesdatenschutzbeauftragter, https://www.bfdi.bund.de/SharedDocs/Publikationen/AusweiskopieAuskunftsersuchen.html.
```

{
    "title": "Wie muss ich mich bei einer Datenschutzanfrage ausweisen?",
    "slug": "ausweispflichten",

>     "date": "2020-04-02T14:00:00+02:00",

Reviewer: ```suggestion
    "date": "2020-05-11T14:00:00+02:00",
```

    "type": "blog",

>     "description": "Wenn Nutzer eine Datenschutzanfrage stellen, werden sie häufig vom Diensteanbieter häufig aufgefordert sich auszuweisen. Grundsätzlich müssen sich Betroffene nur mit solchen Informationen ausweisen, die das Unternehmen für die eindeutige Identifizierung benögtigt.",

Reviewer: ```suggestion
    "description": "Wenn Nutzer_innen eine Datenschutzanfrage stellen, werden sie häufig vom Diensteanbieter häufig aufgefordert sich auszuweisen. Grundsätzlich müssen sich Betroffene nur mit solchen Informationen ausweisen, die das Unternehmen für die eindeutige Identifizierung benögtigt.",
```

    "featured_image": "/img/blog/wie-oft-anfragen.de.jpg",
    "notices": "Titelfoto angepasst nach: „[Letters Write Communication](https://pixabay.com/photos/letters-write-communication-1659715/)“ von [Gerhard Gellinger](https://pixabay.com/users/gellinger-201217/) ([Pixabay-Lizenz](https://pixabay.com/service/license/))",
    "authors": [ "steffengrundke" ],
    "license": "cc-by-4.0"
}

> Unternehmen versuchen immer wieder, unliebsame Datenschutzanfragen von Nutzern abzuwehren. Dies geschieht häufig, indem die Unternehmen von den Nutzern detaillierte Nachweise über ihre Identität verlangen. Teilweise fordern die Unternehmen sogar Ausweiskopien an. Solche Identitätsnachweise sind vor allem dann ein Problem, wenn man sich als Nutzer nicht mit dem richtigen Namen oder nur mit solchen Kontaktinformationen angemeldet hat, aus denen der Anbieter nicht ohne Weiteres auf die Person des Nutzers schließen kann. In diesen Fällen würde der Anbieter durch den Identitätsnachweis weitere persönliche Daten erhalten. Auf der anderen Seite haben die Unternehmen jedoch ein Interesse daran, die Daten nur an den betreffenden Nutzer herauszugeben, um sich nicht selbst eines Datenschutzverstoßes schuldig zu machen.

Reviewer: ```suggestion
Unternehmen versuchen immer wieder, unliebsame Datenschutzanfragen von Nutzer_innen abzuwehren. Dies geschieht häufig, indem die Unternehmen von den Nutzer_innen detaillierte Nachweise über ihre Identität verlangen. Teilweise fordern die Unternehmen sogar Ausweiskopien an. Solche Identitätsnachweise sind vor allem dann ein Problem, wenn man sich als Nutzer_in nicht mit dem richtigen Namen oder nur mit solchen Kontaktinformationen angemeldet hat, aus denen die Anbieter_in nicht ohne Weiteres auf die Person der Nutzer_in schließen kann. In diesen Fällen würde die Anbieter_in durch den Identitätsnachweis weitere persönliche Daten erhalten. Auf der anderen Seite haben die Unternehmen jedoch ein Interesse daran, die Daten nur an die betreffenden Nutzer_innen herauszugeben, um sich nicht selbst eines Datenschutzverstoßes schuldig zu machen.
```

> ## Dienste bei denen sich der Nutzer angemeldet hat

Reviewer: ```suggestion
## Dienste bei denen sich die Nutzer_in angemeldet hat
```

> Grundsätzlich gilt immer, dass der Anbieter dem Nutzer die Ausübung seiner Rechte nicht unmöglich gemacht werden darf. Der Anbieter kann sich nur dann einer Auskunft verweigern, wenn er glaubhaft macht, dass er nicht in der Lage ist, den Betroffenen zu identifizieren, Art. 12 II 2 DSGVO. Identifizierung meint in diesem Kontext, dass der Anbieter sicherstellt, dass es wirklich der Nutzer ist, der für den entsprechenden Account Datenschutzrechte geltend macht. Es ist nicht nötig, dass der Anbieter den Account einer natürlichen Person zuordnen kann. Konkret reicht es, wenn der Nutzer nachweist, dass ihm der Account gehört,[^1] beispielsweise durch die hinterlegte Mailadresse, Benutzername und Kundennummer. In keinem Fall sollten Passwörter weitergegeben werden.

Reviewer: ```suggestion
Grundsätzlich gilt immer, dass die Anbieter_in der Nutzer_in die Ausübung ihrer Rechte nicht unmöglich gemacht werden darf. Die Anbieter_in kann sich nur dann einer Auskunft verweigern, wenn sie glaubhaft macht, dass sie nicht in der Lage ist, den Betroffene_n zu identifizieren, Art. 12 II 2 DSGVO. Identifizierung meint in diesem Kontext, dass die Anbieter_in sicherstellt, dass es wirklich die Nutzer_in ist, die für den entsprechenden Account Datenschutzrechte geltend macht. Es ist nicht nötig, dass die Anbieter_in den Account einer natürlichen Person zuordnen kann. Konkret reicht es, wenn die Nutzer_in nachweist, dass ihr der Account gehört,[^1] beispielsweise durch die hinterlegte Mailadresse, Benutzername und Kundennummer. In keinem Fall sollten Passwörter weitergegeben werden.
```

## Ausweiskopien nur in Ausnahmefällen notwendig

> Ausweiskopien sind hauptsächlich in den Fällen erforderlich, in denen der Nutzer Rechte gegen einen Verantwortlichen geltend machen möchte, mit dem er zuvor noch nicht selbst in Kontakt stand. In diesen Fällen hat der Verantwortliche meist keine andere Möglichkeit den Nutzer zu identifizieren. Klassischerweise sind das Situationen in denen die Rechte gegen eine Auskunftei gerichtet sind. Entgegen häufig anderslautender Gerüchte, ist der Ausweisinhaber berechtigt, eine Kopie seines Personalausweises zu erstellen, § 20 II PAuswG. Auch hier sind nur diejenigen Informationen erforderlich, die es dem Anbieter erlauben, die Person, die den Anspruch geltend macht, eindeutig dem vorhandenen Datensatz zuzuordnen. Nicht erforderlich sind in aller Regel Ausweisnummer, Lichtbild, persönliche Merkmale und Staatsangehörigkeit. Der Anbieter darf die Ausweiskopie nur dazu nutzen, die Identität zu überprüfen. Danach muss die Kopie wieder vernichtet werden.[^2] Es empfiehlt sich, die nicht benötigten Ausschnitte des Ausweises bereits beim Scannen abzudecken. So läuft man später nicht Gefahr, dass Daten aus einem retuschierten Bild doch wieder herstellbar sind. Der verantwortliche Datenverarbeiter ist dazu verpflichtet, den Nutzer in der Wahrnehmung seiner Rechte aktiv zu unterstützen. Daraus folgt, dass der Anbieter den Identitätsnachweis nicht unter einem Medienbruch verlangen kann. Handelt es sich um eine internetbasierte Plattform, dürfen auch die Identitätsnachweise online erbracht werden (z.B. per Mail).

Reviewer: Was mir hier fehlt, ist der klare Hinweis, dass der Verantwortliche nur dann eine Ausweiskopie anfordern darf, wenn "begründete Zweifel" bestehen ([1]). Was genau das heißt, wäre sicher auch spannend auszuführen. Ganz einig sind sich die Behörden da auch nicht ([2]).

[1]: https://www.ldi.nrw.de/mainmenu_Aktuelles/Inhalt/Personalausweis-und-Datenschutz/Datenschutz-und-Personalausweis-2019_07.pdf
[2]: https://www.bfdi.bund.de/SharedDocs/Publikationen/AusweiskopieAuskunftsersuchen.html

Reviewer: Bei Auskunfteien sollte als Identifikation in aller Regel ja genügen, dass die Daten an die der Auskunftei bekannten Adresse gesendet werden.

> Ausweiskopien sind hauptsächlich in den Fällen erforderlich, in denen der Nutzer Rechte gegen einen Verantwortlichen geltend machen möchte, mit dem er zuvor noch nicht selbst in Kontakt stand. In diesen Fällen hat der Verantwortliche meist keine andere Möglichkeit den Nutzer zu identifizieren. Klassischerweise sind das Situationen in denen die Rechte gegen eine Auskunftei gerichtet sind. Entgegen häufig anderslautender Gerüchte, ist der Ausweisinhaber berechtigt, eine Kopie seines Personalausweises zu erstellen, § 20 II PAuswG. Auch hier sind nur diejenigen Informationen erforderlich, die es dem Anbieter erlauben, die Person, die den Anspruch geltend macht, eindeutig dem vorhandenen Datensatz zuzuordnen. Nicht erforderlich sind in aller Regel Ausweisnummer, Lichtbild, persönliche Merkmale und Staatsangehörigkeit. Der Anbieter darf die Ausweiskopie nur dazu nutzen, die Identität zu überprüfen. Danach muss die Kopie wieder vernichtet werden.[^2] Es empfiehlt sich, die nicht benötigten Ausschnitte des Ausweises bereits beim Scannen abzudecken. So läuft man später nicht Gefahr, dass Daten aus einem retuschierten Bild doch wieder herstellbar sind. Der verantwortliche Datenverarbeiter ist dazu verpflichtet, den Nutzer in der Wahrnehmung seiner Rechte aktiv zu unterstützen. Daraus folgt, dass der Anbieter den Identitätsnachweis nicht unter einem Medienbruch verlangen kann. Handelt es sich um eine internetbasierte Plattform, dürfen auch die Identitätsnachweise online erbracht werden (z.B. per Mail).

Reviewer: ```suggestion
Ausweiskopien sind hauptsächlich in den Fällen erforderlich, in denen die Nutzer_in Rechte gegen eine_n Verantwortliche_n geltend machen möchte, mit dem er_sie zuvor noch nicht selbst in Kontakt stand. In diesen Fällen hat die Verantwortliche meist keine andere Möglichkeit die Nutzer_in zu identifizieren. Klassischerweise sind das Situationen in denen die Rechte gegen eine Auskunftei gerichtet sind. Entgegen häufig anderslautender Gerüchte, ist die Ausweisinhaber_in berechtigt, eine Kopie seines_ihres Personalausweises zu erstellen, § 20 II PAuswG. Auch hier sind nur diejenigen Informationen erforderlich, die es der Anbieter_in erlauben, die Person, die den Anspruch geltend macht, eindeutig dem vorhandenen Datensatz zuzuordnen. Nicht erforderlich sind in aller Regel Ausweisnummer, Lichtbild, persönliche Merkmale und Staatsangehörigkeit. Die Anbieter_in darf die Ausweiskopie nur dazu nutzen, die Identität zu überprüfen. Danach muss die Kopie wieder vernichtet werden.[^2] Es empfiehlt sich, die nicht benötigten Ausschnitte des Ausweises bereits beim Scannen abzudecken. So läuft man später nicht Gefahr, dass Daten aus einem retuschierten Bild doch wieder herstellbar sind. Die verantwortliche Datenverarbeiter_in ist dazu verpflichtet, der Nutzer_in in der Wahrnehmung seiner Rechte aktiv zu unterstützen. Daraus folgt, dass die Anbieter_in den Identitätsnachweis nicht unter einem Medienbruch verlangen kann. Handelt es sich um eine internetbasierte Plattform, dürfen auch die Identitätsnachweise online erbracht werden (z.B. per Mail).
```
